conf/battle/items.conf

```diff
@@ -121,3 +121,7 @@ allow_bound_sell: 0x0
 // no = normal refine chances in effect (official/default value)
 // yes = event refine chances in effect
 event_refine_chance: no
+
+// Hide n last characters of player's name with asterisk (*) when the player
+// obtained an item with special broadcast flag.
+broadcast_hide_name: 0
```

Comment:
Shouldn't this be defaulted to 2 to match official behavior?

Reply:
the `2` was set on my import file xD